core/src/main/java/com/linecorp/armeria/client/encoding/ZlibStreamDecoder.java

```diff
@@ -34,34 +35,41 @@
 class ZlibStreamDecoder implements StreamDecoder {
 
     private final EmbeddedChannel decoder;
+    private final ByteBufAllocator alloc;
 
-    ZlibStreamDecoder(ZlibWrapper zlibWrapper) {
+    private boolean withPooledObjects;
+
+    ZlibStreamDecoder(ZlibWrapper zlibWrapper, ByteBufAllocator alloc) {
         decoder = new EmbeddedChannel(false, ZlibCodecFactory.newZlibDecoder(zlibWrapper));
+        this.alloc = alloc;
```

Comment:
Just curious. It seems not used. Did you intend it?

Reply:
Oops, thanks for noticing! Was originally allocating buffers but ended up using the embedded channel's buffer instead, so I need to pass the alloc to it :)